### DISCUSSION TITLE
[Question]: Minimal public chat config example

### DISCUSSION THREAD

**Discussion by h4de5** [danny-avila/LibreChat#2546] [Q&A]
### What is your question?

Is there a minimal example configuration to have a public chat bot, that does not require a registration, does not allow file uploads, does not allow managing of previous chats and does not allow to configure presets etc.

### More Details

What is the best way to achieve this kind of setup? is librechat the right tool for this?

### What is the main subject of your question?

Installation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Some of these features yes, some no.

> that does not require a registration, does not allow file uploads,

LibreChat requires registration, whether it's done by you for the user, or by the user. You can disable file uploads.

> does not allow to configure presets etc.

This is something that is being developed and will release soon.


> does not allow managing of previous chats 

This is not planned but editing the codebase yourself to achieve this wouldn't be too hard. I could see about also letting the admin configure this, but it does not seem like a great need for many users.
